src/display/process/allProcs.go

```diff
@@ -155,6 +155,7 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 
 	previousKey := ""
 	selectedStyle := ui.NewStyle(ui.ColorYellow, ui.ColorClear, ui.ModifierBold)
+	killingStyle := ui.NewStyle(ui.ColorWhite, ui.ColorRed, ui.ModifierBold)
 
 	// updates process list immediately
 	updateProcs := func() {
```

Comment:
@Samyak2 In this function since you're using the `sync.Once` variable, will it really call `updateUI`? Because the once variable is used to render the very first data coming in from the data channel.

Reply:
Apart from this, the `updateProcs()` function is called at the very beginning of the kill section when `K` is pressed. I'm not sure what your reasoning is to call that there. Please let me know?

Reply:
@Gituser143 you are right, it won't actually call `updateUI`. I had copied this part from `case data := <-dataChannel:` and forgot to remove the Once line 😅.
The reasoning behind calling `updateProcs` at the beginning of the kill section was to ensure that the process selected for kill actually exists. But upon further testing, I found that this leads to a different process getting selected for killing.

IMO a better approach would be doing this:
 - After a process is selected for killing, monitor it in every update
 - if the PID still exists, do nothing
 - if the PID does not exist, remove selection